api/liquidate.ts

```diff
@@ -64,9 +66,10 @@ const liquidationHandler = async function () {
         console.log("OK, all positions liquidated.");
 
         return;
-    } catch (err) {
+    } catch (error) {
+        liquidationErrors.inc({ error });
```

Comment:
same with this if the error is diferent then prometheus will consider it a new counter, and most errors will have some part about them different whether its a timestamp or something else